src/video/x11/SDL_x11events.c

```diff
@@ -1377,6 +1377,27 @@ X11_DispatchEvent(_THIS)
                     printf("New _NET_FRAME_EXTENTS: left=%d right=%d, top=%d, bottom=%d\n", data->border_left, data->border_right, data->border_top, data->border_bottom);
                     #endif
                 }
+            } else {
+                char* name_of_atom = X11_XGetAtomName(display, xevent.xproperty.atom);
+
+                if (SDL_strncmp(name_of_atom, "_ICC_PROFILE", sizeof("_ICC_PROFILE")) == 0) {
```

Comment:
I think this was meant to be `sizeof("_ICC_PROFILE") - 1`, otherwise it rejects `_ICC_PROFILE_1` etc.